REFERENCE.md

```diff
@@ -13,7 +13,7 @@
 * [`extlib::sort_by_version`](#extlibsort_by_version): A function that sorts an array of version numbers
 * [`ip_to_cron`](#ip_to_cron): Provides a "random" value to cron based on the last bit of the machine IP address. used to avoid starting a certain cron job at the same time
 * [`random_password`](#random_password): A function to return a string of arbitrary length that contains randomly selected characters.
-* [`resources_deep_merge`](#resources_deep_merge): Deeply merge a "defaults" hash into a "resources" hash like the ones expected by create_resources(). Internally calls the puppetlabs-stdlib f
+* [`resources_deep_merge`](#resources_deep_merge): Deeply merge a "defaults" hash into a "resources" hash like the ones expected by `create_resources()`.  Internally calls the puppetlabs-stdli
```

Comment:
I'm not sure how to make this better and still document the examples etc. Summaries are cut off at 140 chars.

Reply:
I think the internally calls part isn't relevant for the summary so I'd leave that off

Reply:
Some of the example (that we can't use `@example` for) will get sucked into the summary instead?